.dockerignore

```
@@ -0,0 +1,9 @@
+.github/
+.tests/
+.vscode/
+.editorconfig
+.gitattributes
+.gitignore
```

We can also ignore the `.git` directory
```suggestion
.git
```